libs/libc/machine/Kconfig

```diff
@@ -137,6 +137,30 @@ config LIBC_ARCH_ELF_64BIT
 	default n
 	depends on LIBC_ARCH_ELF
 
+config LIBC_PREVENT_KERNEL
```

Comment:
should we change to LIBC_PREVENT_STRING_xxx to limit the scope of exclusion?

Reply:
done!